doczrc.js

```diff
@@ -0,0 +1,47 @@
+export default {
+  base: "/spec/",
+  typescript: true,
+  ignore: ["CODEOFCONDUCT.md", "LICENSE.md", "README.md"],
+  menu: [
```

Comment:
I recall that the other setup auto added to this. Do new files get auto added if running in dev mode? If not do we need to add this to the readme as a note to add stuff?

Reply:
New files will automatically be added, but if you want to control where they appear in the sidebar, we'll have to add them here. Dokz actually did the same thing, but it buried this config in `pages/.app.jsx`.